docs/topics/native/native-definition-file.md

```diff
@@ -0,0 +1,266 @@
+[//]: # (title: Definition file)
+
+Kotlin/Native helps consume C and Objective-C libraries, allowing you to use their functionality in Kotlin.
+A special cinterop tool takes a C or an Objective-C library and generates the corresponding Kotlin bindings,
+so that the library methods can be used in your Kotlin code as usual.
+
+To generate these bindings, each library should have its definition file, usually with the same name as the library.
+This is a property file that describes how exactly the library should be consumed. See the full [list of available properties](#properties).
+
+Here's a general workflow when working with a project:
+
+1. Create a `.def` file describing what to include into bindings.
+2. Use generated bindings in your Kotlin code.
+3. Run the Kotlin/Native compiler to produce the final executable.
+
+## Create and configure a definition file
+
+Let's create a definition file and generate bindings for a C library:
+
+1. In your IDE, select the `src` folder and create a new directory with **File | New | Directory**.
+2. Name new directory **nativeInterop/cinterop**.
+   
+   This is the default convention for `.def` file locations, though it can
+   be overridden in the `build.gradle.kts` file if you use a different location.
+3. Select this new subfolder and create a `png.def` file with **File | New | File**.
+4. Add the necessary properties:
+
+   ```none
+   headers = png.h
+   headerFilter = png.h
+   package = png
+   
+   compilerOpts.linux = -I/usr/include -I/usr/include/x86_64-linux-gnu
+   linkerOpts.osx = -L/opt/local/lib -L/usr/local/opt/png/lib -lpng
+   linkerOpts.linux = -L/usr/lib/x86_64-linux-gnu -lpng
+   ```
+
+   * `headers` is the list of header files to generate Kotlin stubs. You can add multiple files to this entry,
```

Comment:
"that generate Kotlin stubs" or "to generate Kotlin stubs for"?

Reply:
It's the latter